### DISCUSSION TITLE
SellingApiForbiddenException: Access to German Data is Denied

### DISCUSSION THREAD

**Discussion by sktong** [saleweaver/python-amazon-sp-api#509] [Q&A] [answered]
Hi,

I have authorized the data from German market and successfully retrieve the Token. But I end up with the SellingApiForbiddenException. My code is:

credentials = dict(
    refresh_token='refresh_token',
    lwa_app_id='lwa_app_id',
    lwa_client_secret='lwa_client_secret',
    aws_access_key='aws_access_key',
    aws_secret_key='aws_secret_key',
    role_arn = 'role_arn',
    
)
res = ReportsV2(credentials=credentials).get_reports(reportTypes=['GET_V2_SETTLEMENT_REPORT_DATA_FLAT_FILE_V2'],
                                                    )
print(res.payload)

The error message is: SellingApiForbiddenException: [{'message': 'Access to requested resource is denied.', 'code': 'Unauthorized', 'details': ''}]

Can anyone help to solve this issue? Thanks.

**Comment by IP050** [comment, marked as answer]
Try:

from sp_api.base import Marketplaces
ReportsV2(credentials=credentials, marketplace=Marketplaces.DE)

**Reply by sktong** [reply]
Thanks so much！
